src/modules/auth/components/SignUp/FormSignUp.component.tsx

```diff
@@ -22,56 +23,65 @@ export function FormSignUp(): JSX.Element {
     formState: { errors },
   } = useForm<FormInputs>();
 
-  const onSubmit = async (data: FormInputs) => {
-    console.log('Form Data:', data);
+  const onSubmit = (data: FormInputs) => {
+    console.log(data);
   };
-  console.log(errors);
 
   return (
     <>
-      <h2 className="pb-4 text-4xl">{t('auth:signUp')}</h2>
       <form onSubmit={handleSubmit(onSubmit)} className="flex flex-col space-y-4">
-        <Input
+        <TrrInput
           type="text"
           placeholder={t('auth:firstNamePlaceholder')}
+          autoComplete="first-name"
           label={t('auth:firstNameLabel')}
           registerProps={register('firstName', { required: true })}
           error={errors['firstName'] && t(`auth:error:${errors['firstName'].type}`)}
         />
-        <Input
+        <TrrInput
           type="text"
           placeholder={t('auth:lastNamePlaceholder')}
+          autoComplete="family-name"
           label={t('auth:lastNameLabel')}
           registerProps={register('lastName', { required: true })}
           error={errors['lastName'] && t(`auth:error:${errors['lastName'].type}`)}
         />
-        <Input
+        <TrrInput
           type="date"
           placeholder={t('auth:birthdayLabel')}
+          autoComplete="bday"
           label={t('auth:birthdayLabel')}
           registerProps={register('date', { required: true })}
           error={errors['date'] && t(`auth:error:${errors['date'].type}`)}
         />
-        <Input
+        <TrrInput
           type="email"
           placeholder={t('auth:emailPlaceholder')}
-          autoComplete="current-email"
+          autoComplete="email"
           label={t('auth:emailLabel')}
-          registerProps={register('email', { required: true })}
+          registerProps={register('email', {
+            required: true,
+            pattern: {
+              value: emailRegex,
+              message: 'Entered value does not match email format',
+            },
+          })}
           error={errors['email'] && t(`auth:error:${errors['email'].type}`)}
         />
-        <Input
+        <TrrInput
           type="password"
           placeholder={t('auth:passwordPlaceholder')}
-          autoComplete="current-password"
+          autoComplete="new-password"
           label={t('auth:passwordLabel')}
           registerProps={register('password', { required: true, minLength: 8 })}
           error={errors['password'] && t(`auth:error:${errors['password'].type}`)}
         />
-        <Checkbox registerProps={register('terms', { required: true })} className="gap-2">
-          Yes, I accept <a className="cursor-pointer text-primary">privacy policy</a> &{' '}
-          <a className="cursor-pointer text-primary">terms of use</a>.
-        </Checkbox>
+        <TrrCheckbox registerProps={register('terms', { required: true })}>
+          {t('auth:checkBoxString')}{' '}
+          <a className="cursor-pointer text-primary">{t('auth:checkBoxPrivacy')}</a>{' '}
```

Comment:
We should use `Link` component instead of `a` because it is working better with `react-router`... 
But since we don't have privacy & terms pages, we can leave it for now...